composer.json

```diff
@@ -26,5 +28,27 @@
         "psr-4": {
             "RebelCode\\Modular\\": "src"
         }
-    }
+    },
+    "scripts": {
+        "test": "phpunit",
```

Comment:
Composer pushes `vendor/bin` to the top (beginning) of the `PATH` dynamically when executing scripts. This causes the local vendor binaries to be looked up first. From now on, scripts that run vendor binaries should use the command name *only*, without the relative path. So, keep this as is.